project/Dependencies.scala

```diff
@@ -30,7 +31,9 @@ object Dependencies {
     "org.scalatest" %% "scalatest" % ScalaTestVersion % Test) ++ JdbcDrivers.map(_ % Test)
 
   val Migration: Seq[ModuleID] =
-    Seq("com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion).map(_ % Compile) ++ (Seq(
+    Seq(
+      "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion,
+      "com.typesafe" %% "ssl-config-core" % SslConfigVersion).map(_ % Compile) ++ (Seq(
```

Comment:
Akka `2.6.16` (minimum we support) depends on version `0.4.2` of `ssl-config-core`.
By updating `slick-pg` to `0.21.0` a conflict happens on the `scala-parser-combinators`:

```bash
[error] (migration / update) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error]
[error] 	* org.scala-lang.modules:scala-parser-combinators_2.13:2.1.1 (early-semver) is selected over 1.1.2
[error] 	    +- com.github.tminglei:slick-pg_core_2.13:0.21.0      (depends on 2.1.1)
[error] 	    +- com.github.tminglei:slick-pg_2.13:0.21.0           (depends on 2.1.1)
[error] 	    +- com.github.tminglei:slick-pg_circe-json_2.13:0.21.0 (depends on 2.1.1)
[error] 	    +- com.typesafe:ssl-config-core_2.13:0.4.2            (depends on 1.1.2)
```

Specifying a newer version of the `ssl-config-core` library solves the issue and the tests run correctly. However, looking at `akka` they specifically keep the version of this library at `0.4.3` for Scala 2 and only updated to latest version on Scala 3. WDYT @lomigmegard?

Reply:
Found more things about this [here](https://github.com/akka/akka/pull/31252#pullrequestreview-914223000), seems like a workaround.

Reply:
Either way we don't use ssl from akka directly, thus I would not be too worried to update.